neo/Persistence/LevelDB/DbSnapshot.cs

```diff
@@ -47,6 +48,15 @@ public DbSnapshot(DB db)
             ValidatorsCount = new DbMetaDataCache<ValidatorsCountState>(db, options, batch, Prefixes.IX_ValidatorsCount);
             BlockHashIndex = new DbMetaDataCache<HashIndexState>(db, options, batch, Prefixes.IX_CurrentBlock);
             HeaderHashIndex = new DbMetaDataCache<HashIndexState>(db, options, batch, Prefixes.IX_CurrentHeader);
+            StateRootHashIndex = new DbMetaDataCache<HashIndexState>(db, options, batch, Prefixes.IX_CurrentStateRoot,
+            () =>
+            {
+                return new HashIndexState
+                {
+                    Index = 0,
+                    Hash = UInt256.Zero,
```

Comment:
If we want to be network dependant, we could use the Genesis here, what do you think?

Reply:
I want to store, precisely mark root index here, after verify local root with signed root.
I use this for no local root verified.

Reply:
Do you think we should use genesis hash as the initial or empty state root?